### PR TITLE
Fixes #20228: Document only ncf_const variables

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/20228

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

File: src/reference/modules/usage/pages/variables.adoc [hunk 1]
```diff
@@ -541,22 +541,22 @@ There are also more variables available, all documented in https://docs.cfengine
 |Name
 |Description
 
-|`const.dollar`
+|`rudder_const.dollar`
```
Comment (ncharles) [inline src/reference/modules/usage/pages/variables.adoc:544]: i don't have such on variable on my rudder 6.2

**Review by ncharles — CHANGES_REQUESTED** [review]
i don't see what would define these variables

**Comment by ncharles** [comment]
related https://github.com/Normation/ncf/pull/1322/files

**Review by ncharles — CHANGES_REQUESTED** [review]
It should states that these variables appears in 6.2.11 (or better, in 7.0) otherwise people with older version of rudder will have a broken config

**Review by VinceMacBuche — APPROVED** [review]

**Pull request closed without merging**